addons/project_account/models/project_project.py

```diff
@@ -139,7 +139,7 @@ def action_profitability_items(self, section_name, domain=None, res_id=False):
     def _get_domain_aal_with_no_move_line(self):
         """ this method is used in order to overwrite the domain in sale_timesheet module. Since the field 'project_id' is added to the "analytic line" model
         in the hr_timesheet module, we can't add the condition ('project_id', '=', False) here. """
-        return [('account_id', '=', self.analytic_account_id.id), ('move_line_id', '=', False), ('category', '!=', 'manufacturing_order')]
+        return [('account_id', '=', self.analytic_account_id.id), '|', ('move_line_id', '=', False), ('move_line_id.move_type', 'not in', ['out_invoice', 'in_invoice', 'in_refund']), ('category', '!=', 'manufacturing_order')]
```

Comment:
Apparently, Postgres does not manage correctly `not in` with selection field, see 4badf74d

And so, we could try to help our friend, Postgres: 

```suggestion
        return [('account_id', '=', self.analytic_account_id.id), '|', ('move_line_id', '=', False), ('move_line_id.move_type', 'in', ['entry', 'out_refund', 'out_receipt', 'in_receipt']), ('category', '!=', 'manufacturing_order')]
```
🙂